src/embedding/kd.ts

```diff
@@ -253,6 +258,29 @@ export namespace EmbeddedKpiDashboard {
         return getEventType(obj) === GdcKdCommandType.Save;
     }
 
+    /**
+     * Creates a new dashboard from an existing dashboard
+     *
+     * Contract:
+     *
+     * -  if KD saves as new an existing dashboard, the DashboardSaved event will be posted
+     * -  if KD doesn't have an existing dashboard, CommandFailed is posted
```

Comment:
should describe more detail this contract for these case
- duplicate in view mode
- duplicate in edit mode
- duplicate locked dashboard

Reply:
Updated!